Lib9c/Model/Character/ArenaCharacter.cs

```diff
@@ -71,7 +71,7 @@ public int CurrentHP
         public ArenaCharacter(
             ArenaSimulator simulator,
             ArenaPlayerDigest digest,
-            ArenaSimulatorSheets sheets,
```

Comment:
hard-forked action doesn't need sheets anymore?

Reply:
This sheet is renamed to `ArenaSimulatorSheetsV1`.